communication.md

```diff
@@ -0,0 +1,26 @@
+# Communication channels
+
+## Between coordinators
+
+* IM - Telegram
```

Comment:
Telegram is fine for me, but from my experience building open communities, at some point we would like to reach more people to engage them, so maybe Slack (free plan) could be an option to consider

Reply:
I'm using [gitter](https://gitter.im/) and It could be a good slack replacement... It's free and open source :-D

Reply:
@chadell : you mean a public slack channel for conversations between coordinators? Happy to keep that open, so yeah, slack may work... but I'm personally not very reactive to slack (maybe other coordinators have also specific tastes?), and maybe these chats are not so interesting for others (very concrete discussions about organisation?) while for community related topics (knowledge-sharing, Q&As..) I was thinking about something "less IM", like Google Groups.

Reply:
we are experimenting with Slack channel in the NetBCN meetup either for private organization chats and for members interaction, and my experience it's good so far, as we are trying to engage people between event.
In an "open" Slack people understand less responsiveness and also let you create per-topic groups if we like...

Reply:
Nice to know! Yeah, Im open to that, and nothing needs to be written into stone... so we can try, and then adapt/change/open/close channels that are not working... Other views?

Reply:
I prefer "Telegram" for coordination stuff and "Slack/Gitter/IM" for community chat.

Reply:
Given we already have it, Im keeping the telegram for coordination, but Slack AND Google groups for community (we can experiment and react to the usage we see). See latest version of the page